src/dams_mcda/ui.R

```diff
@@ -1132,12 +1088,12 @@ ui <- shinyUI(fluidPage(
 				DT::dataTableOutput("Dam1RawTable"),
 		        downloadButton("DownloadDam1RawTable", "Download Table", style="width:100%;"),
 				HTML(
-					"<br><b>Results Interpretation</b> for Table 2. This table displays the raw data values we collected and/or calculated/generated through our research for each decision criterion and alternative. You may \
-					remember seeing these data when you clicked on the link for the data for this dam during the preference elicitation in Step 3. We include the raw data values again here to help make the MCDA calculations \
-					more transparent, so you can clearly see what goes into the final calculation that produces Figures 1 and 2 above. In addition, you can use this table to sort decision alternatives in ascending or descending order\
-					in each column by clicking on the arrow next to the column header . Note: fish survival values shown here are discrete, but in reality, the values are network-dependent and would be impacted by upstream or downstream\
-					changes. They are presented here as the average of a range of possible values for this dam, depending on what happens at other dams. This interaction between decisions at other dams and these decision criteria are\
-					modeled in the multi-objective optimization that leads to the final Map Recommendation.<br>"
+				  "<br><b>Results Interpretation</b> for Table 2. This table displays the raw data values we collected and/or calculated/generated through our research for each decision criterion and alternative. You may \
```

Comment:
Need to take out Lines  1095 (starting with ""depending on what happens at other dams...") to 1096.

Reply:
1095 is mid sentence